llvm/include/llvm/TargetParser/Triple.h

```diff
@@ -276,7 +276,7 @@ class Triple {
     Callable,
     Mesh,
     Amplification,
-
+    OpenCL,
```

Comment:
I wonder why we need this addition. This is not mentioned in the description.

Reply:
Some cases like https://github.com/llvm/llvm-project/blob/main/clang/test/CodeGenOpenCL/amdgpu-alignment.cl#L3